### PR TITLE
PLAT-17000: Apply rAF into Control.applyStyle

### PR DESCRIPTION
**Pull request by ybsung**
Issue
------
: We need to reduce JS execution time to handle touchmove and to update DOM element while scrolling a list.
 
Fix
------
: I applied rAF into not enyo/ScrollMath.animate.fn() but enyo/Control.applyStyle() and called enyo/ScrollMath.animate.fn() directly when handing "touchmove" event.

Enyo-DCO-1.1-Signed-off-by: YB Sung <yb.sung@lge.com>

### CODE REVIEW

File: src/ScrollMath.js [hunk 1]
```diff
@@ -464,17 +462,14 @@ module.exports = kind(
 			y0 = this.y;
 			x0 = this.x;
 		});
-		this.job = animation.requestAnimationFrame(fn);
+		fn();
```
Comment (webOS101) [inline src/ScrollMath.js]: If we're going to invoke this immediately why don't we save a bindSafely call and just remove the wrapping function?

File: src/touch.js [hunk 2]
```diff
@@ -8,7 +8,10 @@ var
 
 var
 	Dom = require('./dom'),
-	Job = require('./job');
+	Job = require('./job'),
+	Control = require('./Control');
+
+var rAF = global.requestAnimationFrame;
```
Comment (webOS101) [inline src/touch.js]: We have a wrapper around rAF to prevent issues where it doesn't exist on the platform:  animation.requestAnimationFrame.

**Comment by ybsung** [comment]
Thanks, @webOS101.

I fixed the following issues that @webOS101 commented.
- Inserted the code in fn() into animate()
- Used animation.requestAnimationFrame instead of using window.requestAnimationFrame directly

And I merged the latest code in garnet-perf-exp into this PR.

**Comment by 0x64** [comment]
After we test this experimental branch, we must refine code.

**Pull request merged**